client/components/public/imports/HomeEarlyBird.jsx

```diff
@@ -1,52 +1,72 @@
 import React, { Component, PropTypes } from 'react';
```

Comment:
I wonder if `HomeEarlyBird` is no longer an appropriate file name for this file? Since it includes the main registration steps? idk. not a big deal for now, just a thought